phpunit.xml

```diff
@@ -15,10 +15,17 @@
     </testsuites>
 
     <logging>
-        <log type="coverage-html" target="tests/coverage/"/>
+        <log type="coverage-clover" target="build/logs/clover.xml"/>
+        <log type="coverage-php" target="build/logs/coverage.cov"/>
+        <log type="coverage-text" target="build/logs/coverage.txt" showUncoveredFiles="false"/>
+        <log type="coverage-html" target="test/reports/coverage/" showUncoveredFiles="true"/>
```

Comment:
Right! That does not mean it's not important to make it right :)
Actually, if you make the change, we'll see immediately if it works as I enabled coveralls for the repo already. If Github shows the coverall report, then we're good!